web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss

```diff
@@ -934,3 +934,15 @@ input::-webkit-textfield-decoration-container {
     max-width: 100%;
   }
 }
+
+// hides the alert status text in the alert component
+.usa-screen-reader {
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  word-wrap: normal;
+}
```

Comment:
There is an existing `.visuallyhidden` class that is the same as this (save a line or two). Let's re-use the existing class instead of adding a new one.

We could rename it. That involves touching a bunch of files. Looks like 4 in the theme folder, and `benefitSearch.js` in the modules\custom folders.